oak_functions/proto/abi.proto

```diff
@@ -46,6 +46,10 @@ enum OakStatus {
   ERR_STORAGE_ITEM_NOT_FOUND = 4;
   // Error when running the TensorFlow model, due to bad input tensor.
   ERR_BAD_TENSOR_FLOW_MODEL_INPUT = 5;
+  // Error when extension handle is invalid.
+  ERR_INVALID_EXTENSION_HANDLE = 6;
```

Comment:
At this level I don't think there should be a mention of extensions, this is just a transport for performing invocations over some handle. The fact that there is an extension or a service or another Wasm module or something else should not appear at this level. Also we may want to use this for other communications, e.g. between trusted and untrusted runtime, or between Oak and other systems, etc. I suggest just renaming to ERR_INVALID_HANDLE.